static/templates/settings/convert_demo_organization_form.hbs

```diff
@@ -0,0 +1,33 @@
+<div id="convert-demo-organization-form">
+    {{#unless email}}
+    <form class="form-horizontal email-form">
+        <div class="input-group new_email_container">
+            <label class="demo_organization_form_email" for="email"><b>{{t 'Email' }}</b></label>
+            <input type="text"
+              class="email required"
+              placeholder="{{t 'Enter your email address' }}"
+              id="email"
+              name="email"
+              autofocus
+              required />
+        </div>
+    </form>
+    {{/unless}}
+
+    {{#if email}}
+    <p>You can convert this demo organization to a permanent Zulip organization. All users and message history will be preserved.</p>
+    <form class="form-horizontal subdomain-setting">
+        <div class="input-group new_subdomain_container">
+            <label class="realm_string_id" for="string_id"><b>{{t "Organization URL" }}</b></label>
+            <input type="text"
+              autocomplete="off"
+              name="string_id"
+              placeholder="{{t 'subdomain' }}"
+              value="{{ realm_subdomain }}"
+              required />
+            <label for="string_id" class="domain_label">.{{ realm_domain }}</label>
+            <div id="help-text">The URL users will use to access the new organization.</div>
```

Comment:
I occasionally read URL as an adjective describing 'users' here, which doesn't make sense. I feel like adding 'that' would just help clarify this sentence ...

> The URL that users will use to access the new organization.

Reply:
I found this phrase odd too when I saw it, but it's the same as the wording we use on the org creation page, and we should keep the two places consistent unless there's good reason not to.

The "users will use" phrasing bothers me, so if we are going to change it, we should probably brainstorm in #design to try to get to wording we actually like.

Reply:
This was removed from the new organization form in commit 899f535b39cc3a91818e2cb658b0f9ee5baf5772.